src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -3504,6 +3507,17 @@ public bool ShouldReleaseTransactionBecauseFlushIsWaiting(IndexingStatsScope sta
                 }
             }
 
+            if (Configuration.MangedAllocationsBatchLimit != null)
+            {
+                var currentManagedAllocations = new Size(GC.GetAllocatedBytesForCurrentThread(), SizeUnit.Bytes);
+                var diff = currentManagedAllocations - _initialManagedAllocations;
+                if (diff > Configuration.MangedAllocationsBatchLimit.Value)
+                {
+                    stats.RecordMapCompletedReason($"Reached managed allocations limit ({Configuration.MangedAllocationsBatchLimit.Value}). Allocated {diff} in current batch");
```

Comment:
ToString on 'Configuration.MangedAllocationsBatchLimit.Value' and 'diff' yields human readable string?

Reply:
yes, those are `Size`.